files/runScanner.sh

```diff
@@ -1,18 +1,27 @@
 #!/bin/bash
-echo "setting up user & logfile:"
-if [[ -z ${UID} ]]; then
-	UID=1000
-fi
-if [[ -z ${GID} ]]; then
-	GID=1000
+
+mandatory_vars=(NAME MODEL IPADDRESS)
+missing=0
+for var in "${mandatory_vars[@]}"
+do
+	if [[ -z "${!var}" ]]; then
+		echo "missing mandatory variable: $var"
+		missing=1
+	fi
+done
+if [[ $missing -eq 1 ]]; then
+	echo "exiting"
+	exit 1
 fi
-groupadd --gid $GID NAS
-adduser $USERNAME --uid $UID --gid $GID --disabled-password --force-badname --gecos ""
+
+echo "setting up logfile"
+
 mkdir -p /scans
 chmod 777 /scans
 touch /var/log/scanner.log
-chown $USERNAME /var/log/scanner.log
+chmod 777 /var/log/scanner.log
 env > /opt/brother/scanner/env.txt
```

Comment:
With the following line, this should no longer be needed, I guess?!

Reply:
yep